js/layout-javascript/small-card-code.js

```diff
@@ -325,18 +325,19 @@ DynamicList.prototype.attachObservers = function() {
       }
 
       var _that = $(this);
+      var $el = $(event.target);
```

Comment:
`$(event.target)` and `$(this)` are the same, which means `_that` and `$el` are the same. Pick one, and whichever you go with, make sure the variable is prefixed with `$`.

Reply:
@tonytlwu fixed